### ISSUE TITLE
Evaluation of Communication Plan

### ISSUE THREAD

**Issue by nurlandadashov02** [bounswe/bounswe2022group3#141]
### Task:
As stated in issue #132 :  Evaluation of Communication Plan will be written as a subsection of Evaluation of Deliverables. Current Communication plan can be found [here](https://github.com/bounswe/bounswe2022group3/wiki/Communication-Plan).

### Place To Put The Documentation:
Milestone 1 Report (After its done there will be a link here)

### Deadline: 
14.04.2022 21.00

**Comment by nurlandadashov02** [comment]
All in all I think our team used some tools for communicating effectively. A problem would be brought up to the attention of the team, and the team would immediately act to solve the problem. We used Zoom for our weekly meetings. Zoom is a great platform, the however 40-minute restriction made it harder to use. One of the first things we did as a group was to set up a Slack workspace. We use slack to communicate while working on some tasks. We created appropriate channels to keep conversations about different topics apart.  We use Whatsapp for urgent communication and just chatting. We additionally use Github issues to communicate about a specific issue.

**Comment by hsuleerkul** [comment]
Looks good!

**Comment by ersoykadir** [comment]
> All in all I think our team used some tools for communicating effectively. A problem would be brought up to the attention of the team, and the team would immediately act to solve the problem. We used Zoom for our weekly meetings. Zoom is a great platform, the however 40-minute restriction made it harder to use. One of the first things we did as a group was to set up a Slack workspace. We use slack to communicate while working on some tasks. We created appropriate channels to keep conversations about different topics apart.  We use Whatsapp for urgent communication and just chatting. We additionally use Github issues to communicate about a specific issue.

I think small adjustments can be made such as:
> All in all, I think our team used the chosen tools for communication effectively. Whenever a problem would brought up to the attention of the team, they would immediately act to solve the problem. We used Zoom for our weekly meetings. Zoom is a great platform; however, 40-minute restriction made it harder to use ...

**Comment by nurlandadashov02** [comment]
@ersoykadir I agree with you. 

*Latest version:*

All in all, I think our team used the chosen tools for communication effectively. Whenever a problem would be brought up to the attention of the team, they would immediately act to solve the problem. We used Zoom for our weekly meetings. Zoom is a great platform; however, the 40-minute restriction made it harder to use. One of the first things we did as a group was to set up a Slack workspace. We use slack to communicate while working on some tasks. We created appropriate channels to keep conversations about different topics apart.  We use Whatsapp for urgent communication and just chatting. We additionally use Github issues to communicate about a specific issue.
